src/app/tests/suites/certification/Test_TC_WNCV_4_3.yaml

```diff
@@ -47,20 +47,19 @@ tests:
               maxValue: 10000
 
     - label:
-          "1b: If (PA_LF & LF) TH reads CurrentPositionLiftPercentage from DUT"
+          "1b 1c: If (PA_LF & LF) TH reads CurrentPositionLiftPercentage from
+          DUT + assert CurrentPositionLiftPercent100ths/100 equals
+          CurrentPositionLiftPercentage"
+      disabled: true
```

Comment:
Why disabled?  The new codegen bits are not being exercised if this is disabled....

Reply:
Per offline discussion: the codegen is correct when the test is enabled, but it fails because of bugs in the window covering cluster that are in the process of being fixed.